api/feedback/v1/feedback.proto

```diff
@@ -94,14 +92,8 @@ message SubmitFeedbackRequest {
   // TODO: remove if we decide to record feedback only when a user clicks the submit button
   string id = 1 [ (validate.rules).string = {min_bytes : 1} ];
```

Comment:
i'd actually bump the length requirement up here, maybe even use exactly and be prescriptive about how ids should be generated on the client

Reply:
good idea, thx! set the len to exactly 36 chars a a uuid is 32 chars with 4 hyphens.
https://www.npmjs.com/package/uuid, https://www.ietf.org/rfc/rfc4122.txt

should i set this length on the db field as well?

cc @jdslaugh